### ISSUE TITLE
version number broken on macOS

### ISSUE THREAD

**Issue by core-code** [aws-ew-dev/ElasticWolf#113]
i've downloaded the latest release 5.1.7 but when selecting it in the Finder it claims to be version 5.0.3

**Comment by john-aws** [comment]
Hi, can you clarify where you are seeing version number 5.0.3?

**Comment by core-code** [comment]
<img width="714" alt="screen shot 2018-10-19 at 10 37 06" src="https://user-images.githubusercontent.com/1261954/47207295-2c2f1c80-d38b-11e8-8938-038f2f1e1dd1.png">

**Comment by john-aws** [comment]
@core-code Thanks for the detailed response. We will take a look. My understanding is that this does not actually impact the function of ElasticWolf and you are actually running the latest code.

**Comment by core-code** [comment]
yes actual functionality is not impacted, but there are several reasons having the right version in the app would be beneficial:
• users would be able to check the version number in the Finder in case they don't remember which version they actually downloaded (like with any other app)
• users could use third-party updater apps to check for updates if the app had correct versions
• if multiple versions are installed concurrently the system fails to launch the newest one as it should do if the actual version numbers are incorrect

> We will take a look. 

awesome, thanks!
